_ml-commons-plugin/connectors.md

```diff
@@ -440,17 +490,76 @@ POST /_plugins/_ml/connectors/_create
 }
 ```
 
-The `credential` parameter contains the following options reserved for `aws-sigv4` authentication:
+The `credential` parameter contains the following options reserved for `aws_sigv4` authentication:
 
 - `access_key`: Required. Provides the access key for the AWS instance.
 - `secret_key`: Required. Provides the secret key for the AWS instance.
 - `session_token`: Optional. Provides a temporary set of credentials for the AWS instance.
 
-The `paramaters` section requires the following options when using `aws-sigv4` authentication:
+The `parameters` section requires the following options when using `aws_sigv4` authentication:
 
 - `region`: The AWS Region in which the AWS instance is located.
 - `service_name`: The name of the AWS service for the connector.
 
+### Cohere
+
+The following example request creates a standalone Cohere connection:
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+    "name": "Cohere Connector: embedding",
+    "description": "The connector to cohere embedding model",
+    "version": 1,
+    "protocol": "http",
+    "credential": {
+        "cohere_key": "..."
+    },
+    "actions": [
+        {
+            "action_type": "predict",
+            "method": "POST",
+            "url": "https://api.cohere.ai/v1/embed",
+            "headers": {
+                "Authorization": "Bearer ${credential.cohere_key}"
+            },
+            "request_body": "{ \"texts\": ${parameters.prompt}, \"truncate\": \"END\" }"
+        }
+    ]
+}
+```
+{% include copy-curl.html %}
+
+## Building blueprints
+
+All connectors consist a JSON blueprint created by ML developers to allow administrators and data scientists to make connections between OpenSearch and an AI service or model serving technology. 
+
+The following configuration options are **required** in order to build a connector blueprint. These settings can be used for both external and local connectors.
+
+| Field | Data type | Description |
+| :---  | :--- | :--- |
+| `name` | String | The name of the connector. |
+| `description` | String | A description of the connector. |
+| `version` | Integer | The version of the connector. |
+| `protocol` | String | The protocol for the connection. For AWS services such as Amazon SageMaker and Amazon Bedrock, use `aws_sigv4`. For all other services, use `http`. |
+| `parameters` | JSON array | The default connector parameters, including `endpoint` and `model`.  Any parameters indicated in this field can be overrided by parameters made in a predict request. |
```

Comment:
The parameters is not a Json array, it's a Json object which is a collection of [name–value pairs](https://en.wikipedia.org/wiki/Attribute%E2%80%93value_pair) where the names (also called keys) are strings.